src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -214,7 +221,7 @@ describe('HmctsGlobalHeaderComponent', () => {
       active: false,
       roles: ['roleC']
     }];
-    component.ngOnChanges(changesMock);
+    await component.ngOnChanges(changesMock);
```

Comment:
@johnbenjamin-hmcts - see previous comment on `await`.

Reply:
Thank you @Daniel-Lam - I have now reverted the changes.